.github/workflows/release.yml

```diff
@@ -68,16 +68,18 @@ jobs:
         with:
           password: ${{ secrets.TEST_PYPI_API_TOKEN }}
           repository-url: https://test.pypi.org/legacy/
+          skip-existing: true
```

Comment:
C&P for visibility, in case there's additional considerations I missed.

> Pro tip: try to avoid enabling this setting where possible. If you have steps for publishing to both PyPI and TestPyPI, consider only using it for the latter, having the former fail loudly on duplicates.

If we're scoped to only run the workflow on release, risk is low?

Reply:
I think the risk is low in our case. This "protip" makes me scratch my head — it's really difficult to put together automations when actions aren't idempotent.